src/thor/multimodal.cc

```diff
@@ -390,9 +390,15 @@ bool MultiModalPathAlgorithm::ExpandForward(GraphReader& graphreader,
         } else {
           if (pred.tripid() > 0) {
             // tripId > 0 means the prior edge was a transit edge and this
-            // is an "in-station" transfer. Add a small transfer time and
-            // call GetNextDeparture again if we cannot make the current
-            // departure.
+            // is an "in-station" transfer.
+            // This mean changing vehicle (unboard, wait, board again).
+            // So first check if the pedestrian costing allow traversing
+            // this node.
+            if (!pc->Allowed(nodeinfo)) {
+              continue;
+            }
```

Comment:
If `wheelchair_boarding` was set to `false` and access mask keeped on the stop node, a normal pedestrian costing won't allow wheelchair users to "walk" (unboard and board)  through this node.